Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY controllers/ controllers/
 COPY internal/ internal/
 COPY models/ models/
 COPY pkg/ pkg/
-COPY restapi/ restapi/
+#COPY restapi/ restapi/
 COPY watchers/ watchers/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -mod=vendor -a -o manager main.go
```

Comment:
perhaps a different name than manager? maybe flotta-api-server?

Reply:
That's the original Dockerfile for the operator; the new one is [here](https://github.com/project-flotta/flotta-operator/blob/4464d12711666e04248a1976e87f10285824e5e4/build/httpapi/Dockerfile).